org.citra.citra.json

```
@@ -0,0 +1,72 @@
+{
+    "app-id": "org.citra.citra",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "citra-qt",
+    "rename-desktop-file": "citra.desktop",
+    "rename-icon": "citra",
+    "build-options": {
+        "env": {
+            "CI": "1",
+            "GITHUB_ACTIONS": "1",
+            "GITHUB_REPOSITORY": "citra-emu/citra-nightly",
+            "GIT_TAG_NAME": "nightly-1694"
+        }
+    },
+    "finish-args": [
+        "--device=all",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--share=ipc",
+        "--filesystem=host:ro"
+    ],
+    "modules": [
+        {
+            "name": "citra",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DENABLE_QT_TRANSLATION=ON",
+                "-DCITRA_ENABLE_COMPATIBILITY_REPORTING=ON",
+                "-DENABLE_COMPATIBILITY_LIST_DOWNLOAD=ON",
+                "-DENABLE_FFMPEG_VIDEO_DUMPER=ON",
+                "-DENABLE_FDK=ON"
+            ],
+            "cleanup": [
+              "/share/man",
+              "/share/pixmaps"
+            ],
+            "post-install": [
+                "install -Dm644 ../appdata.xml /app/share/appdata/org.citra.citra.appdata.xml",
+                "desktop-file-install --dir=/app/share/applications ../dist/citra.desktop",
+                "echo 'StartupWMClass=citra-qt' >> /app/share/applications/citra.desktop",
+                "install -Dm644 ../dist/citra.svg /app/share/icons/hicolor/scalable/apps/citra.svg",
+                "install -Dm644 ../dist/icon.png /app/share/icons/hicolor/512x512/apps/citra.png",
+                "mv /app/share/mime/packages/citra.xml /app/share/mime/packages/org.citra.citra.xml",
+                "sed 's/citra/org.citra.citra/g' -i /app/share/mime/packages/org.citra.citra.xml"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/citra-emu/citra-nightly.git",
+                    "tag": "nightly-1694",
+                    "commit": "e6c479f497612a0ae1c00ad9c85d5af715495d46",
+                    "disable-shallow-clone": true
+                },
+                {
+                    "type": "file",
+                    "path": "appdata.xml"
```

```suggestion
                    "path": "org.citra.citra.metainfo.xml"
```
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html